test/cel-validation/envoyproxy_test.go

```diff
@@ -1615,6 +1615,24 @@ func TestEnvoyProxyProvider(t *testing.T) {
 			},
 			wantErrors: []string{},
 		},
+		{
+			desc: "valid: image set with digest, imageRepository not set",
```

Comment:
can we also add a test for `envoyproxy/envoy@sha256:da99c47f08546492d19973920dc76334c592f59ad5b732a514320d959db9fa40`
which is the more common way for pulling by digest